src/index.js

```diff
@@ -500,6 +504,43 @@ const indy = {
     return [credDefId, JSON.parse(credDef)]
   },
 
+  async buildGetRevocRegDefRequest(submitterDid: Did | null, revocRegDefId: string): Promise<LedgerRequest> {
+    if (Platform.OS === 'ios') {
+      throw new Error(`Unsupported operation! Platform: ${Platform.OS}`)
+    }
+    return JSON.parse(await IndySdk.buildGetRevocRegDefRequest(submitterDid, revocRegDefId))
+  },
+
+  async parseGetRevocRegDefResponse(getRevocRegResponse: LedgerRequestResult): Promise<GetRevocRegDefResponse> {
```

Comment:
I haven't found `GetRevocRegDefResponse` type definition. It should be something like `Promise<[revocRegDefId, revocRegDef]`

You can copy-paste `revocRegDef` from https://github.com/DefinitelyTyped/DefinitelyTyped/blob/2ac94cb7ad65f90e03f31179151243086072110f/types/indy-sdk/index.d.ts#L415

As I'm thinking of it right now it could be nice to somehow utilize that types from definitely type in the future 🤔

Reply:
I am 100% behind being able to use the types from the DefinitelyTyped source to have a single source of truth vs the current duplication.

Reply:
See https://github.com/hyperledger/indy-sdk-react-native/issues/12. Should be doable if we find the time :)